deployment/ansible/group_vars/all

```diff
@@ -30,6 +30,7 @@ postgis_package_version: "2.3.1+dfsg-1.pgdg14.04+1"
 # azavea.docker
 # 1.11.* to match AWS ECS
 docker_version: 1.11.*
+docker_options: "--storage-opt dm.basesize=20G"
```

Comment:
Hmmm. We don't. Docker sets up the default volume pool mapped to `/var/lib/docker/volumes`. Can you point me to the docs that would indicate what you describe? Definitely willing to play around with this more, although in the case of Vagrant, we're still limited to the default 40Gb VM volume unless we want to mount docker volumes to a shared folder, which would incur the performance hit and probably isn't worth it for this project.